application-tour-ui/src/main/resources/TourCode/WebHomeSheet.xml

```diff
@@ -39,7 +39,7 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{include reference="AppWithinMinutes.LiveTableViewSheet"/}}
+  <content>{{include reference='AppWithinMinutes.LiveTableViewSheet' /}}
```

Comment:
Not useful. Unlike velocity or groovy, I don't think wiki macros make any difference between simple and double quotes.